camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java

```diff
@@ -233,7 +238,16 @@ public void run() throws Exception {
 		Job job = createJob(props);
 		if (getLog4jConfigure(job)) {
 			DOMConfigurator.configure("log4j.xml");
-		}
+    }
+
+    if (Boolean.parseBoolean(props.getProperty(SCHEMA_REGISTRY_CONNECTIVITY_CHECK))) {
```

Comment:
What do you mean exactly? https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html#getProperty(java.lang.String) returns `String`s, need to cast to `Boolean` for the conditional.